jboss-eap-cd-openshift-launch/added/standalone-openshift.xml

```diff
@@ -169,8 +169,8 @@
                 </managed-scheduled-executor-services>
             </concurrent>
             <default-bindings context-service="java:jboss/ee/concurrency/context/default"
-                              datasource="##DEFAULT_DATASOURCE##"
-                              jms-connection-factory="##DEFAULT_JMS##"
+                              <!-- ##DEFAULT_DATASOURCE## -->
```

Comment:
Note this is historical cruft, it was really readdded by the commit after this revert. I'll verify after merge, but it looks fine in my tip.

Reply:
When I merge your branch (or a branch created from pull/9/head from jboss-container-images) I end up with this in the file rather than the correct stuff. So I'll approve this assuming you'll correct it.

Reply:
Yeah, this is annoying, I'll fix.